src/app/common/normalizeWH.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Функция normalize100vh устанавливает 100vh для доступного к просмотру вюпорта
+ * тем самым приводя для всех браузеров высоту к одному виду и адапптирует так
+ * что в браузерах с полосами которые сверху и снизу сайт не заходит под них
+ */
+export function normalize100vh() {
```

Comment:
Мы не используем ключевое слово `function`.

```suggestion
export const watchHeight() {
```